### PR TITLE
The xeno hybrid refurbishing.

### LINKED ISSUE #8771: Xenomorph teeth marking doesn't appear on/is below head styles
## Reproduction:

1 - Character Preferences, have a Xenomorph species character. Make sure to have selected a Xeno head. 
2 - Augments+, select Xeno Head on Markings.

The teeth doesn't show on any other head style but the None/Default one. It might be because the head style is on a higher layer than the markings.

With Xeno Warrior head, with teeth marking on
image

Without Xeno head, with teeth marking off
image

Without Xeno head, with teeth marking on
image

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request

This is for the like, 2 xeno hybrid players, have prettier sprites. While I have to get everything else.
Tin, cleans up the xeno hybrid sprites, makes them look generally better along with fixing the palette. Be warned these give a more concise color than the original xeno hybrid sprites, please tweak your color to make sure they look correct. The xeno heads were specifically tweaked to fit with the xeno hybrid species so they look off with other species, don't try to use them on other ones. I've also redone the tail wag animation to be a bit more fluid.

Swapped the Hollywood head with a 'netted' variant which has a net pattern imprinted on the dome.

Also fixes a year long bug of the teeth appearing under the head. Please add that in the augments menu under the head tab.

fixes #8771

## How This Contributes To The Skyrat Roleplay Experience

Prettier sprites.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
General overlook of the sprites.
Xenos

Tail wag.

https://user-images.githubusercontent.com/81479835/201504073-72fea640-4c34-4e78-89ab-25069d9a35db.mp4

</details>

## Changelog

:cl:
imageadd: Xenohybrids have had a general sprite facelift, please tweak any xeno hybrid character's colors as they are more accurate.
fix: Xenohybrid teeth now actually appear on xenohybrids, please select them from the augment menu under head markings.
/:cl:

### CODE REVIEW

**Comment by RatFromTheJungle** [comment]
sheeesh...

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/xeno.dm [hunk 1]
```diff
@@ -50,8 +50,8 @@
 	icon_state = "royal"
 
 /datum/sprite_accessory/xenohead/hollywood
-	name = "Hollywood"
-	icon_state = "hollywood"
+	name = "Nethead"
+	icon_state = "net"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/xeno.dm]: If it's no longer called hollywood it probably shouldn't be still using the `/datum/sprite_accessory/xenohead/hollywood` typepath.

**Review by GoldenAlpharex — DISMISSED** [review]
Also, I think you can remove the sprites for the alternative digi legs in that xeno parts dmi, I don't think they're used anywhere anymore.

That being said, some pretty nice sprites indeed.

File: modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/xeno.dm (hunk 1 above)
Comment (Ebin-Halcyon) [reply modular_skyrat/modules/customization/modules/mob/dead/new_player/sprite_accessories/xeno.dm]: Oh whoop, that slipped through the cracks. I'll get that fixed real quick.

**Review by Zonespace27 — APPROVED** [review]

**Pull request merged**